examples/wearable/UIComponents/contents/navcontrols/indexscrollbar/indexScrollbar.js

```diff
@@ -5,7 +5,8 @@
 		listviewElement = document.getElementById("list1"),
 		isCircle = tau.support.shape.circle,
 		scroller,
-		indexScrollbar;
+		indexScrollbar,
+		listview;
```

Comment:
Can we initialize it here?
 listview = tau.engine.getBinding(listviewElement);

Reply:
We cannot initialize anything in this way before "pageshow" event.